packages/baby-jubjub/rollup.iife.config.ts

```diff
@@ -35,13 +33,5 @@ export default {
         }
     ],
     external: [],
-    plugins: [
-        typescript({ tsconfig: "./build.tsconfig.json" }),
-        commonjs(),
-        nodeResolve({
-            preferBuiltins: true
-        }),
-        nodePolyfills({ include: null }),
-        cleanup({ comments: "jsdoc" })
-    ]
+    plugins: [typescript({ tsconfig: "./build.tsconfig.json" }), commonjs(), cleanup({ comments: "jsdoc" })]
```

Comment:
It shouldn't need `commonjs`, and it can be merged with the `rollup.config.ts` I think.